src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,53 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a list of tasks.
+ */
+public class TaskList extends ArrayList<Task> {
```

Comment:
TaskList inherits from ArrayList which makes the entire codebase really clean

Reply:
+1! Beautiful code!!